src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -48,6 +48,9 @@ void setUserStore(UserStore userStore) {
     this.userStore = userStore;
   }
 
+  void resetUserStore() {
+    this.userStore = null;
```

Comment:
What is this function for?

Reply:
Originally I was trying to add a log out feature so I forgot to delete it.